packages/wdio-browserstack-service/src/log4jsAppender.ts

```diff
@@ -0,0 +1,22 @@
+import logReportingAPI from './logReportingAPI.js'
+
+const BSTestOpsLogger = new logReportingAPI({})
+
+//Disabling eslint here as there params can be used later
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+function BSTestOpsLog4JSAppender(layout: Function, timezoneOffset: any): Function {
+    return (loggingEvent: any) => {
+        BSTestOpsLogger.log({
+            level: loggingEvent.level ? loggingEvent.level.levelStr : null,
+            message: loggingEvent.data ? loggingEvent.data.join(' ') : null
+        })
+    }
+}
+
+export const configure = (config: any, layouts: any): Function => {
+    let layout = layouts.colouredLayout
+    if (config.layout) {
+        layout = layouts.layout(config.layout.type, config.layout)
+    }
+    return BSTestOpsLog4JSAppender(layout, config.timezoneOffset)
```

Comment:
Can we not call this "log4js"? It suggest you are using this package which you aren't.

Reply:
We are not using `log4js` in our code, we are just defining an appender that users can use to configure in their log4js configuration. So we don't need the log4js package for this